src/data/SearchIndex.test.js

```
@@ -0,0 +1,72 @@
+        model1.string = 'abc def';
+        const model2 = SimpleModelWithSearchIndexFactory();
+        model2.string = 'uvw xyz';
+        const index = new SearchIndex(SimpleModelWithSearchIndex, {
+            [model1.id]: model1.toSearchData(),
+            [model2.id]: model2.toSearchData(),
+        });
+        const result = index.search('abc');
+
+        test('returns the expected model', () => {
+            expect(result).toStrictEqual([
+                new SearchResult(SimpleModelWithSearchIndex.fromData(model1.toSearchData()), 0.693),
```

Biome (1.9.4)

[error] Prefer constants from the standard library.

Unsafe fix: Use Math.LN2 instead.


(lint/suspicious/noApproximativeNumericConstant)